TransformationSystem/Client/TaskManager.py

```diff
@@ -333,6 +335,116 @@ def __init__( self, transClient = None, logger = None, submissionClient = None,
     self.destinationPlugin_o = None
 
   def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGroup = '', ownerDN = '' ):
+
+    bulkFlag = self.opsH.getValue( "Transformations/BulkSubmission", False )
+    if bulkFlag:
+      result = self.__prepareTransformationTasksBulk( transBody, taskDict, owner, ownerGroup, ownerDN )
+    else:
+      result = self.__prepareTransformationTasks( transBody, taskDict, owner, ownerGroup, ownerDN )
+    return result
+
+  def __prepareTransformationTasksBulk( self, transBody, taskDict, owner = '', ownerGroup = '', ownerDN = '' ):
+
+    if ( not owner ) or ( not ownerGroup ):
+      res = getProxyInfo( False, False )
+      if not res['OK']:
+        return res
+      proxyInfo = res['Value']
+      owner = proxyInfo['username']
+      ownerGroup = proxyInfo['group']
+
+    if not ownerDN:
+      res = getDNForUsername( owner )
+      if not res['OK']:
+        return res
+      ownerDN = res['Value'][0]
+
```

Comment:
Lines 348 - 360 are also at the beginning of ` def __prepareTransformationTasks` and can be put in `def prepareTransformationTasks`